conf/broker.conf

```diff
@@ -1,3 +1,14 @@
+[general]
+## Force a token refresh on login with the local password. This has the
+## effect that the user is only allowed to log in if they still have
+## the required permissions in the identity provider.
+## If this option is set to false (the default), the token is only
+## refreshed if the provider is reachable.
+## Important: If this option is enabled, authd users won't be able to
+## log in with the local password if the identitiy provider is
+## unreachable (e.g. because there is no working internet connection).
+#force_token_refresh_on_login = false
```

Comment:
To the reviewer(s): I felt like the option didn't really fit in the existing sections (`oidc` and `users`) so I added this new `general` section. I'm open for other proposals.

Reply:
Yeah, I agree with this, naming of the section can be discussed, but this one looks good to me

Reply:
@didrocks are you also fine with this new `general` section?

Reply:
now that we have `provider` in the option name (`force_provider_authentication`), we could also consider moving it to the `oidc` section, because it's related to the OIDC provider.

Reply:
I think it’s fine to move to the oidc section to avoid creating a new one, wdyt?